Frameworks/include/CGGraphicBufferImage.h

```diff
@@ -20,8 +21,9 @@
 
 class CGGraphicBufferImage : public __CGImage {
 public:
-    CGGraphicBufferImage(DWORD width, DWORD height, surfaceFormat fmt);
-    CGGraphicBufferImage(DWORD width, DWORD height, surfaceFormat fmt, DisplayTexture* nativeTexture, DisplayTextureLocking* locking);
+    CGGraphicBufferImage(int width, int height, surfaceFormat fmt);
+    CGGraphicBufferImage(__CGSurfaceInfo* surfaceInfo);
```

Comment:
>__CGSurfaceInfo [](start = 25, length = 15)

const _CGSurfaceInfo* (since you are not changing it).  Altertnatively const __CGSurfaceInfo& so you could do this like new CGGraphicsBufferImage(_CGInitSurface(...)).